appside/nav/nav.js

```diff
@@ -23,6 +23,7 @@ class Nav {
         }
         this.current_frame = model.get_frame('next');
         this.render();
+        this.model.log('app-start');
```

Comment:
Wondering if storing 'app-end' might be helpful as well.
(Helps us figure out how long an activity lasts + possibly if user truly read our content.)

Reply:
Yes! End time is important.

#21 specifies the following time stamps that we want to log (as soon as the corresponding frames are implemented)
```
            pre-measure-start: time,
            pre-selfreport-start: time,
            mood-induction-start: time,
            app-start: time,
            post-measure-start: time,
            post-survey-start: time,
            end: time,
```
My thought is that in general, the end of one section is the same time as the beginning of the next section. So `app-end` will be the same as `post-measure-start`.

Taking a closer look, I've logged `app-start` in the wrong place -- this will log it on the very first frame of the model, even if there are frames before the app.

What I have yet to figure out is the mechanism for logging each of these at the correct time. Some options:
* View -- frames know when they are rendered. Certain frames log that timestamp. Caveat: business logic in the view.
* Model -- Model logs it upon calls to next_frame. Caveat: a view could theoretically fetch frames without rendering them to the user.
* Nav -- View reports to Nav when it renders itself. Nav logs the timestamp. Caveat: not sure this is any better than option 1.

Reply:
possible ideas
* make the logger static. Caveat: it's stateful. Stateful objects shouldn't be static. Is this option just an excuse for not having to pass it in to each frame?
* pass logger into each frame.

Reply:
For now, this line is temporarily removed to allow commit to master.